rustsec/Cargo.toml

```diff
@@ -24,10 +24,10 @@ toml = "0.7"
 url = { version = "2", features = ["serde"] }
 
 # optional dependencies
-tame-index = { version = "0.9.3", default-features = false, features = ["git", "sparse", "native-certs"], optional = true }
+tame-index = { version = "0.9.8", default-features = false, features = ["git", "sparse", "native-certs"], optional = true }
```

Comment:
@Shnatsel the latest twist in this saga is that `tame-index` v0.9.8 and v0.9.9 have been yanked. Looks like we need to bump to v0.10?

Reply:
Yeah, I asked for them to be yanked because they are semver-breaking. We do need to bump to v0.10 now which would be gix 0.61 I believe (it's yet another gix bump)